README.md

```diff
@@ -6,6 +6,8 @@ Submit translations here: https://crowdin.com/project/joinmastodon
 
 ## Development
 
+Mastodon's homepage requires Node 16 LTS and Yarn for development.
```

Comment:
Perhaps something like this would be better?

```suggestion
To build the site, you'll need to have Node.js version 16 installed, which you can [download from their website](https://nodejs.org/download/release/v16.20.0/) or install via tools like `fnm` or `nvm`. For dependency management, `yarn` is used, it should be automatically be installed when you install Node.js, if it's not available you can use `corepack enable` and `corepack prepare yarn@1` to install it.
```